scalafix-cli/src/main/scala/scalafix/internal/v1/MainOps.scala

```diff
@@ -198,7 +199,13 @@ object MainOps {
       }
 
     if (!args.args.autoSuppressLinterErrors) {
```

Comment:
Can we move this whole if block into a separate method `handlePathDiagnostics`? The method is already big enough.